components/index.ts

```diff
@@ -8,6 +8,7 @@ export * from './PaginationSection'
 export * from './SearchBar'
 export * from './ServiceItem'
 export * from './ServicesListItem'
+export * from './ShareLink'
 export * from './SortMenu'
 export * from './TaxonomySection'
-export * from './ShareLink'
+export * from './theme'
```

Comment:
Think it makes more sense to have `theme.ts` at the top level of `components` rather than under `Layout`